app/views/kaui/subscriptions/_form.html.erb

```diff
@@ -7,15 +7,27 @@
         <%= hidden_field_tag :base_product_name, @base_product_name %>
     <% elsif @subscription.product_category == 'BASE' %>
         <div class="form-group">
-          <%= f.label :external_key, 'Key', :class => 'col-sm-2 control-label' %>
+          <%= f.label :bundle_external_key, 'Bundle Key', :class => 'col-sm-2 control-label' %>
           <div class="col-sm-10">
-            <%= f.text_field :external_key, :class => 'form-control', :id => 'external_key' %>
-            <div class="subscription_external_key_invalid_msg text-danger">
+            <%= f.text_field :bundle_external_key, :class => 'form-control', :id => 'bundle_external_key' %>
+            <div class="subscription_bundle_external_key_invalid_msg text-danger">
               Warning! External key already in use.
             </div>
           </div>
         </div>
     <% end %>
+
+    <div class="form-group">
+      <%= f.label :subscription_external_key, 'Subscription Key', :class => 'col-sm-2 control-label' %>
+      <div class="col-sm-10">
+        <%= f.text_field :external_key, :class => 'form-control', :id => 'subscription_external_key' %>
```

Comment:
No change required in `Kaui::SubscriptionsController#create` (both keys are correctly created)? It might just work because of our conventions.

Reply:
Yep, precisely..